gadgets/src/binary_number.rs

```diff
@@ -176,6 +176,7 @@ where
         value: &T,
     ) -> Result<(), Error> {
         for (&bit, &column) in value.as_bits().iter().zip(&self.config.bits) {
+            region.name_column(|| format!("GADGETS_binary_number_{:?}", column), column);
```

Comment:
Same as previous comment

Reply:
fixed in [1b5aa41](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1152/commits/1b5aa4193380414a8378031399be7a6eddfac11a)